spec/index.bs

```diff
@@ -8152,17 +8162,21 @@ GPUQueue includes GPUObjectBase;
                                 {{GPUTextureUsage/RENDER_ATTACHMENT}} and {{GPUTextureUsage/COPY_DST}}.
                             - |textureDesc|.{{GPUTextureDescriptor/dimension}} must be {{GPUTextureDimension/"2d"}}.
                             - |textureDesc|.{{GPUTextureDescriptor/sampleCount}} must be 1.
-                            - |textureDesc|.{{GPUTextureDescriptor/format}} must be one of the following:
+                            - |textureDesc|.{{GPUTextureDescriptor/format}} must be one of the following
+                                formats (which all support {{GPUTextureUsage/RENDER_ATTACHMENT}} usage):
+                                - {{GPUTextureFormat/"r8unorm"}}
+                                - {{GPUTextureFormat/"r16float"}}
+                                - {{GPUTextureFormat/"r32float"}}
+                                - {{GPUTextureFormat/"rg8unorm"}}
+                                - {{GPUTextureFormat/"rg16float"}}
+                                - {{GPUTextureFormat/"rg32float"}}
```

Comment:
Do we define what values rg textures get from 1/2/3/4-channel inputs?

Reply:
Added a paragraph about this.